one_fm/one_fm/custom/task.json

```diff
@@ -557,7 +623,189 @@
    "property": "field_order",
    "property_type": "Data",
    "row_name": null,
-   "value": "[\"github_sync_id\", \"subject\", \"project\", \"issue\", \"type\", \"is_routine_task\", \"routine_erp_document\", \"routine_erp_docname\", \"color\", \"is_group\", \"is_template\", \"column_break0\", \"status\", \"priority\", \"task_weight\", \"parent_task\", \"completed_by\", \"completed_on\", \"custom_assigned_to\", \"sb_timeline\", \"exp_start_date\", \"expected_time\", \"start\", \"column_break_11\", \"exp_end_date\", \"progress\", \"duration\", \"is_milestone\", \"sb_details\", \"description\", \"sb_depends_on\", \"depends_on\", \"depends_on_tasks\", \"sb_actual\", \"act_start_date\", \"actual_time\", \"column_break_15\", \"act_end_date\", \"sb_costing\", \"total_costing_amount\", \"total_expense_claim\", \"column_break_20\", \"total_billing_amount\", \"sb_more_info\", \"review_date\", \"closing_date\", \"column_break_22\", \"department\", \"company\", \"lft\", \"rgt\", \"old_parent\", \"email_sender\", \"auto_repeat\", \"template_task\"]"
+   "value": "[\"github_sync_id\", \"subject\", \"project\", \"custom_project_type\", \"issue\", \"type\", \"is_routine_task\", \"routine_erp_document\", \"routine_erp_docname\", \"color\", \"is_group\", \"is_template\", \"column_break0\", \"status\", \"priority\", \"task_weight\", \"parent_task\", \"completed_by\", \"completed_on\", \"custom_assigned_to\", \"sb_timeline\", \"exp_start_date\", \"expected_time\", \"start\", \"column_break_11\", \"exp_end_date\", \"progress\", \"duration\", \"is_milestone\", \"sb_details\", \"description\", \"sb_depends_on\", \"depends_on\", \"depends_on_tasks\", \"sb_actual\", \"act_start_date\", \"actual_time\", \"column_break_15\", \"act_end_date\", \"sb_costing\", \"total_costing_amount\", \"total_expense_claim\", \"column_break_20\", \"total_billing_amount\", \"sb_more_info\", \"review_date\", \"closing_date\", \"column_break_22\", \"department\", \"company\", \"lft\", \"rgt\", \"old_parent\", \"email_sender\", \"auto_repeat\", \"template_task\"]"
+  },
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "creation": "2024-07-12 08:42:29.698748",
+   "default_value": null,
+   "doc_type": "Task",
+   "docstatus": 0,
+   "doctype_or_field": "DocField",
+   "field_name": "is_milestone",
+   "idx": 0,
+   "is_system_generated": 1,
+   "modified": "2024-12-19 16:32:24.835641",
+   "modified_by": "Administrator",
+   "module": null,
+   "name": "Task-is_milestone-in_list_view",
+   "owner": "j.poil@one-fm.com",
+   "parent": null,
+   "parentfield": null,
+   "parenttype": null,
+   "property": "in_list_view",
+   "property_type": "Check",
```

Comment:
These changes are from where ?

Reply:
@pjamsheer , most likely from exporting, it has been fixed